### ISSUE TITLE
Tree keyboard navigation not working with drag&drop

### ISSUE THREAD

**Issue by majugurci** [primefaces/primereact#1183]
**I'm submitting a ...**
```
[x] bug report
[ ] feature request
[ ] support request
```

**Codesandbox Case**
https://codesandbox.io/s/primereact-keyboard-navigation-bug-zsk7z

**Current behavior**
When tree has drag&drop enabled keyboard navigation is not working. It is throwing errors in console (Firefox: element.children[0] is undefined, Chrome: Cannot read property 'focus' of undefined).

**Expected behavior**
It should work.

* **React version:**
16.12.0

* **PrimeReact version:**
3.4.0

* **Browser:**
Firefox, Chrome

**Comment by fylip97** [comment]
We have the same problem with PrimeReact version 4.2.1

**Comment by DumData** [comment]
Still an issue in 7.1.0

**Comment by melloware** [comment]
Yes it was not fixed in 7.1.0 that is why the ticket is still open.

**Comment by melloware** [comment]
This will be fixed when PrimeVue accessibility issues are merged in as PrimeVue does not have this issue.

**Comment by reza-ebrahimi** [comment]
any progress on this issue?

**Comment by habubey** [comment]
The solution to this problem will come with accessibility features. Which we'll start on this quarter.
Issue tracker is used for defects only as part of our commitment to quality and continuous improvement in all areas. Enhancements are collected as valuable community feedback and managed internally so moving this enhancement ticket to our internal project management backlog.
